mistralrs-core/src/engine/mod.rs

```diff
@@ -413,6 +413,7 @@ impl Engine {
                             format!("Stop sequence '{s:?}' encodes to multiple tokens when it should only encode to 1.").into(),
                         ))
                         .unwrap();
+                    return;
```

Comment:
I decide to use this error to test the MR and realized it was crashing the server when the request finished because it tried to send a message to a closed connection.

Reply:
Yes, absolutely. That should be done.